src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -3,53 +3,12 @@ import React, { useState } from "react";
 import styled from "styled-components";
 
 import Colors from "../design-system/Colors";
-import iconEditSrc from "../design-system/icons/ic_edit.svg";
```

Comment:
Looks like we have a couple of outdated imports the linter caught that we can remove now.